### DISCUSSION TITLE
Traduções

### DISCUSSION THREAD

**Discussion by RatoX** [frontendbr/forum#438] [Perguntas e Respostas]
**N**egads,

O que vocês estão usando para tradução de contexto em sistemas grandes no frontend?

**Comment by eduardojmatos** [comment]
@RatoX você diz tradução de idiomas? Se sim, geralmente uso o I18n, seja bom Rails ou JS. 

Geralmente soluções no server side são melhores em termos de SEO.

**Comment by RatoX** [comment]
E quando tu tem um SPA full js servido pelo S3 da Amazon  e se comunicando por API? 

Estou desenvolvendo uma plataforma fechada então SEO não é um foco.

**Comment by eduardojmatos** [comment]
Hum saquei! Se é SPA, presumo que o render de view é no client né? Nesse caso tem a lib do i18n pra JS. Veja se o framework que vc tá usando tem algum adapter pra i18n e manda ver! :)

**Comment by thulioph** [comment]
Já utilizei o [i18n](http://i18next.com/) com render no client e sempre uso quando preciso, bem fácil e tranquilo. +1

**Comment by felipefialho** [comment]
Deu certo @RatoX ?

**Comment by williangd** [comment]
Pessoal que trabalha com i18n no front.. Usam alguma ferramenta para abrir os JSONs e traduzir? 

Os arquivos serão enviados para outras pessoas traduzirem. Em outro projeto onde as traduções ficam em arquivos .PO, eles usam o Poedit para facilitar a tradução. 

Estou procurando uma ferramenta nesse sentido. Vi que tem a BabelEdit que faz isso e é paga.

Edit: Acabei de achar outra ferramenta que é free: https://electronjs.org/apps/i18n-manager

**Comment by kavalcante** [comment]
@williangd Estamos estudando o [Crowdin](https://crowdin.com/), ouvi em um podcast do PodProgramar e na RD eles utilizam ele.

**Comment by ninetails** [comment]
Queria usar esse [cara](http://i18n-tag.kolmer.net/) mas nunca tive oportunidade.
